.gitignore

```diff
@@ -8,3 +8,5 @@ texHeaders.bin
 *.biprivatekey
 Thumbs.db
 CHANGELOG.md
+.hephaestus.json
```

Comment:
What's this?

Reply:
Special config file for [Hephaestus](https://github.com/ArmaAchilles/Hephaestus).
It doesn't have to be in source control as it has full sys paths. I personally use it to quickly start the game and build using armake.

Reply:
Local gitignore?

Add HEMTT support btw. :D

Reply:
I'll add CI support after this PR and then I'll start adding HEMTT support.

Reply:
I'm probably not going to be the only one using it, so I'll just leave it for now.